docs/specviz/plugins.rst

```diff
@@ -103,6 +103,11 @@ show the fitted value of each parameter rather than the initial value, and
 will additionally show the standard deviation uncertainty of the fitted
 parameter value if the parameter was not set to be fixed to the initial value.
 
+Parameter values for each fitting run are stored in the plugin table.  
```

Comment:
```suggestion
Parameter values for each fit are stored in the plugin table.  
```

Reply:
I agree "fitting run" is a bit awkward, but I worry "fit" suggests that they are tied specifically to the output model object, which could be misleading.  Each time "calculate fit" is clicked, a new entry is added.... and if you overwrite the model, you still have a history of the old entry.

Reply:
I know what you mean, but I still think the verb to describe "every time you click 'calculate fit'" should be "fit".